### DISCUSSION TITLE
Attention:  filepicker.exe

### DISCUSSION THREAD

**Discussion by EspritIT** [aclap-dev/video-downloadhelper#412] [Help and Q&A]
### Your question

Hi, since yesterday afternoon the Antivirus has been reporting the presence of "filepicker.exe" which is located in the folder:
"C:\Program Files\DownloadHelper CoApp\"

Yesterday he detected it and placed it in quarantine.

Today after installing the "Supplementary App" (vdhcoapp-windows-x86_64-installer.exe) on "Video DownloadHelper
Version 8.2.2.8a1 (beta)" and the antivirus found the file again.
(see attachment) 

At this moment I have reloaded the supplementary app but the download does not work, the videos remain in the "queued" state

Solutions?

Thanks
![Virus](https://github.com/aclap-dev/video-downloadhelper/assets/150854822/15ba2b3b-5aa8-46fd-9524-1b8d7dea1f8c)

### Addon & CoApp version and details

"Supplementary App" (vdhcoapp-windows-x86_64-installer.exe) 
"Video DownloadHelper Version 8.2.2.8a1 (beta)"

**Comment by jcfrog** [comment]
Hi,
It's a false positive: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-and-Antivirus

**Reply by EspritIT** [reply]
![storico](https://github.com/aclap-dev/video-downloadhelper/assets/150854822/4a99297d-599c-437d-b6b1-55c4a252d253)

MS Defendere is active. Avira doesn't block my installation. He finds it for me when he does a full scan of my PC every week. And before yesterday he never found it. (as you can see from the attached file). From today VideoDownLoader doesn't work, it puts all the videos you want to download in the "queued" state. Your events don't seem like a coincidence to me.

**Reply by GrampaWildWilly** [reply]
Think about the world of software.  Software vendors put out updates all the time.  Do you think every antivirus program knows about every new program that is released every day?  This is why antivirus programs offer a way to tell the antivirus program that any given program is actually NOT a virus, that the antivirus program should NOT quarantine the program.  The antivirus software vendors are relying on their users, meaning YOU, to tell their antivirus programs about new releases of SAFE software.  How many years have you been using VDH?  Aren't you convinced yet that VDH is safe?  Tell your antivirus program that this filepicker.exe program is safe.  Make an exception for it in the settings of your antivirus program.  Either that or deprive yourself of the use of a program you know to be safe & wait for some unknown time until your antivirus vendor gets around to fixing their antivirus program to stop flagging this totally safe component of VDH.

The same explanation applies to firewall software.  Antivirus & firewall software is just that: software.  It is not sacred.  It is as prone to bugs & shortcomings as any other software.  Accept that & move on.
